kedro-datasets/kedro_datasets/snowflake/snowpark_dataset.py

```diff
@@ -4,7 +4,7 @@
 from copy import deepcopy
 from typing import Any, Dict
 
-import snowflake.snowpark as sp
+import snowflake.snowpark as sp  # pylint: disable=import-error
```

Comment:
Is this needed? Does CircleCI throw this error or it's from local? It shouldn't throw any import-error here.

Reply:
ah yeah ooppss its a local error so i just removed it. Thanks Nok